pkg/v1/cli/command/core/repo.go

```diff
@@ -104,6 +106,7 @@ var addRepoCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		log.Successf("Successfully added repository %s", pluginRepo)
```

Comment:
start the message with a small letter "successfully" to be consistent with other messages.